aten/src/ATen/native/cuda/CuFFTPlanCache.h

```diff
@@ -229,10 +265,18 @@ class CuFFTConfig {
       // by assuming base_istride = base_ostride = 1.
       //
       // See NOTE [ cuFFT Embedded Strides ] in native/cuda/SpectralOps.cu.
+#ifdef __HIP_PLATFORM_HCC__
+      int sizes = *signal_sizes.data();
```

Comment:
Don't really need to create a new variable. Can simply pass signal_sizes.data().

Reply:
Actually we do. signal_sizes.data() returns a `long long*` and rocFFT needs a `int*`